src/test/kotlin/no/nav/familie/ef/sak/integration/Testdata.kt

```diff
@@ -0,0 +1,119 @@
+package no.nav.familie.ef.sak.no.nav.familie.ef.sak.integration
+
+import no.nav.familie.ef.sak.integration.dto.pdl.*
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+/**
+ * TODO
+ *
+ * @author Bent J. Lorentzen
+ */
+object Testdata {
```

Comment:
Kom på en sak.. Trengs denne egentlige? Hadde det vært tilsrekkelig å kun kjøre reflection på klasse-objektet? Istedet for en instans

Reply:
Jeg brukte veldig mye tid på å prøve å få det til, men det går ikke å finne typene til listene uten et de er instansiert på grunn av type erasure.